R/sf.R

```diff
@@ -457,13 +457,14 @@ CoordSf <- ggproto("CoordSf", CoordCartesian,
         x_labels <- scale_x$labels
       }
 
-      if (is.character(x_labels)) {
+      # all labels need to be temporarily stored as character vectors,
+      # but expressions need to be parsed afterwards
+      if (is.character(x_labels) || is.factor(x_labels)) {
```

Comment:
Could this just be `needs_parsing[graticule$type == "E"] <- is.character(x_labels) || is.factor(x_labels)` ?

Reply:
It needs a not in front, but otherwise yes, thanks!